### ISSUE TITLE
audiostream does not play in chrome

### ISSUE THREAD

**Issue by androidwiltron** [SussexLearningSystems/peakaboo#44]
I suspect this is a chrome thing however, the audiostream from the CA will not load fully in chrome.
it will partially buffer then stop.
tried enabling auto-play in chrome with no change

firefox and safari seem OK

**Comment by ppettit** [comment]
can confirm this is an issue in chrome, i guess something has changed there. if i remember all the code does is set the mp3 url as the src of an audio element.

**Comment by androidwiltron** [comment]
yep thats correct, nothing special. needs a bit of chrome debugging i think. must be a policy change how it handles media coming from another source. still looking into it
